packages/codec/package.json

```diff
@@ -29,7 +29,7 @@
     "@truffle/compile-common": "^0.7.32",
     "big.js": "^6.0.3",
     "bn.js": "^5.1.3",
-    "cbor": "^5.1.0",
```

Comment:
Same here re minor version bump - was that intentional?

Reply:
See above.